Amplify/DevMenu/Logging/PersistentLogWrapper.swift

```diff
@@ -7,46 +7,52 @@
 
 import Foundation
 
+/// Class that wraps another `Logger` and saves the logs in memory
 class PersistentLogWrapper: Logger {
     var logLevel: LogLevel
 
     var wrapper: Logger
-    var logs: String?
+
+    /// Array of `LogEntry` containing the history of logs
+    private var logHistory: [LogEntryItem] = [LogEntryItem]()
```

Comment:
nit: `[LogEntryItem]()` can be expressed as `[]` since the type was explicit declared along with the variable.